build/openshift/DockerfileWithAppProtectForPlus

```diff
@@ -0,0 +1,133 @@
+ARG GOLANG_CONTAINER=golang:latest
```

Comment:
I suggest placing that file into the appprotect folder, so the sake of putting all app protect related Dockerifiles in one location.  The name can be  `DockerfileWithAppProtectForPlusForOpenShift`